rank=03 L00: Traceback (most recent call last):
rank=03 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=03 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=03 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=03 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=03 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=03 L06:     f(args)
rank=03 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=03 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=03 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=03 L10:     return _run_module_code(code, init_globals, run_name,
rank=03 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=03 L12:     _run_code(code, mod_globals, init_globals,
rank=03 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=03 L14:     exec(code, run_globals)
rank=03 L15:   File "get_me_coupling.py", line 91, in <module>
rank=03 L16:     structure.get_potential_energy()
rank=03 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=03 L18:     energy = self._calc.get_potential_energy(self)
rank=03 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=03 L20:     return self.get_property(name, atoms)
rank=03 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=03 L22:     self.calculate(atoms, [name], system_changes)
rank=03 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=03 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=03 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=03 L26:     yield from self.scf.irun(
rank=03 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=03 L28:     raise KohnShamConvergenceError(
rank=03 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
rank=06 L00: Traceback (most recent call last):
rank=06 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=06 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=06 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=06 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=06 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=06 L06:     f(args)
rank=06 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=06 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=06 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=06 L10:     return _run_module_code(code, init_globals, run_name,
rank=06 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=06 L12:     _run_code(code, mod_globals, init_globals,
rank=06 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=06 L14:     exec(code, run_globals)
rank=06 L15:   File "get_me_coupling.py", line 91, in <module>
rank=06 L16:     structure.get_potential_energy()
rank=06 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=06 L18:     energy = self._calc.get_potential_energy(self)
rank=06 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=06 L20:     return self.get_property(name, atoms)
rank=06 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=06 L22:     self.calculate(atoms, [name], system_changes)
rank=06 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=06 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=06 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=06 L26:     yield from self.scf.irun(
rank=06 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=06 L28:     raise KohnShamConvergenceError(
rank=06 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 6): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=08 L00: Traceback (most recent call last):
rank=08 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=08 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=08 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=08 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=08 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=08 L06:     f(args)
rank=08 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=08 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=08 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=08 L10:     return _run_module_code(code, init_globals, run_name,
rank=08 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=08 L12:     _run_code(code, mod_globals, init_globals,
rank=08 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=08 L14:     exec(code, run_globals)
rank=08 L15:   File "get_me_coupling.py", line 91, in <module>
rank=08 L16:     structure.get_potential_energy()
rank=08 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=08 L18:     energy = self._calc.get_potential_energy(self)
rank=08 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=08 L20:     return self.get_property(name, atoms)
rank=08 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=08 L22:     self.calculate(atoms, [name], system_changes)
rank=08 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=08 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=08 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=08 L26:     yield from self.scf.irun(
rank=08 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=08 L28:     raise KohnShamConvergenceError(
rank=08 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 8): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=12 L00: Traceback (most recent call last):
rank=12 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=12 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=12 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=12 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=12 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=12 L06:     f(args)
rank=12 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=12 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=12 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=12 L10:     return _run_module_code(code, init_globals, run_name,
rank=12 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=12 L12:     _run_code(code, mod_globals, init_globals,
rank=12 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=12 L14:     exec(code, run_globals)
rank=12 L15:   File "get_me_coupling.py", line 91, in <module>
rank=12 L16:     structure.get_potential_energy()
rank=12 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=12 L18:     energy = self._calc.get_potential_energy(self)
rank=12 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=12 L20:     return self.get_property(name, atoms)
rank=12 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=12 L22:     self.calculate(atoms, [name], system_changes)
rank=12 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=12 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=12 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=12 L26:     yield from self.scf.irun(
rank=12 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=12 L28:     raise KohnShamConvergenceError(
rank=12 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 12): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=16 L00: Traceback (most recent call last):
rank=16 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=16 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=16 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=16 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=16 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=16 L06:     f(args)
rank=16 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=16 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=16 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=16 L10:     return _run_module_code(code, init_globals, run_name,
rank=16 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=16 L12:     _run_code(code, mod_globals, init_globals,
rank=16 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=16 L14:     exec(code, run_globals)
rank=16 L15:   File "get_me_coupling.py", line 91, in <module>
rank=16 L16:     structure.get_potential_energy()
rank=16 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=16 L18:     energy = self._calc.get_potential_energy(self)
rank=16 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=16 L20:     return self.get_property(name, atoms)
rank=16 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=16 L22:     self.calculate(atoms, [name], system_changes)
rank=16 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=16 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=16 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=16 L26:     yield from self.scf.irun(
rank=16 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=16 L28:     raise KohnShamConvergenceError(
rank=16 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 16): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=19 L00: Traceback (most recent call last):
rank=19 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=19 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=19 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=19 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=19 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=19 L06:     f(args)
rank=19 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=19 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=19 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=19 L10:     return _run_module_code(code, init_globals, run_name,
rank=19 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=19 L12:     _run_code(code, mod_globals, init_globals,
rank=19 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=19 L14:     exec(code, run_globals)
rank=19 L15:   File "get_me_coupling.py", line 91, in <module>
rank=19 L16:     structure.get_potential_energy()
rank=19 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=19 L18:     energy = self._calc.get_potential_energy(self)
rank=19 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=19 L20:     return self.get_property(name, atoms)
rank=19 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=19 L22:     self.calculate(atoms, [name], system_changes)
rank=19 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=19 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=19 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=19 L26:     yield from self.scf.irun(
rank=19 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=19 L28:     raise KohnShamConvergenceError(
rank=19 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 19): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=23 L00: Traceback (most recent call last):
rank=23 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=23 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=23 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=23 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=23 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=23 L06:     f(args)
rank=23 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=23 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=23 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=23 L10:     return _run_module_code(code, init_globals, run_name,
rank=23 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=23 L12:     _run_code(code, mod_globals, init_globals,
rank=23 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=23 L14:     exec(code, run_globals)
rank=23 L15:   File "get_me_coupling.py", line 91, in <module>
rank=23 L16:     structure.get_potential_energy()
rank=23 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=23 L18:     energy = self._calc.get_potential_energy(self)
rank=23 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=23 L20:     return self.get_property(name, atoms)
rank=23 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=23 L22:     self.calculate(atoms, [name], system_changes)
rank=23 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=23 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=23 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=23 L26:     yield from self.scf.irun(
rank=23 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=23 L28:     raise KohnShamConvergenceError(
rank=23 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 23): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=00 L00: Traceback (most recent call last):
rank=00 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=00 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=00 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=00 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=00 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=00 L06:     f(args)
rank=00 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=00 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=00 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=00 L10:     return _run_module_code(code, init_globals, run_name,
rank=00 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=00 L12:     _run_code(code, mod_globals, init_globals,
rank=00 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=00 L14:     exec(code, run_globals)
rank=00 L15:   File "get_me_coupling.py", line 91, in <module>
rank=00 L16:     structure.get_potential_energy()
rank=00 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=00 L18:     energy = self._calc.get_potential_energy(self)
rank=00 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=00 L20:     return self.get_property(name, atoms)
rank=00 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=00 L22:     self.calculate(atoms, [name], system_changes)
rank=00 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=00 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=00 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=00 L26:     yield from self.scf.irun(
rank=00 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=00 L28:     raise KohnShamConvergenceError(
rank=00 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 0): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=01 L00: Traceback (most recent call last):
rank=01 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=01 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=01 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=01 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=01 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=01 L06:     f(args)
rank=01 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=01 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=01 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=01 L10:     return _run_module_code(code, init_globals, run_name,
rank=01 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=01 L12:     _run_code(code, mod_globals, init_globals,
rank=01 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=01 L14:     exec(code, run_globals)
rank=01 L15:   File "get_me_coupling.py", line 91, in <module>
rank=01 L16:     structure.get_potential_energy()
rank=01 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=01 L18:     energy = self._calc.get_potential_energy(self)
rank=01 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=01 L20:     return self.get_property(name, atoms)
rank=01 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=01 L22:     self.calculate(atoms, [name], system_changes)
rank=01 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=01 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=01 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=01 L26:     yield from self.scf.irun(
rank=01 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=01 L28:     raise KohnShamConvergenceError(
rank=01 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
rank=04 L00: Traceback (most recent call last):
rank=04 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=04 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=04 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=04 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=04 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=04 L06:     f(args)
rank=04 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=04 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=04 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=04 L10:     return _run_module_code(code, init_globals, run_name,
rank=04 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=04 L12:     _run_code(code, mod_globals, init_globals,
rank=04 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=04 L14:     exec(code, run_globals)
rank=04 L15:   File "get_me_coupling.py", line 91, in <module>
rank=04 L16:     structure.get_potential_energy()
rank=04 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=04 L18:     energy = self._calc.get_potential_energy(self)
rank=04 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=04 L20:     return self.get_property(name, atoms)
rank=04 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=04 L22:     self.calculate(atoms, [name], system_changes)
rank=04 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=04 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=04 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=04 L26:     yield from self.scf.irun(
rank=04 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=04 L28:     raise KohnShamConvergenceError(
rank=04 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
rank=05 L00: Traceback (most recent call last):
rank=05 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=05 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=05 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=05 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=05 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=05 L06:     f(args)
rank=05 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=05 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=05 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=05 L10:     return _run_module_code(code, init_globals, run_name,
rank=05 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=05 L12:     _run_code(code, mod_globals, init_globals,
rank=05 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=05 L14:     exec(code, run_globals)
rank=05 L15:   File "get_me_coupling.py", line 91, in <module>
rank=05 L16:     structure.get_potential_energy()
rank=05 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=05 L18:     energy = self._calc.get_potential_energy(self)
rank=05 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=05 L20:     return self.get_property(name, atoms)
rank=05 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=05 L22:     self.calculate(atoms, [name], system_changes)
rank=05 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=05 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=05 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=05 L26:     yield from self.scf.irun(
rank=05 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=05 L28:     raise KohnShamConvergenceError(
rank=05 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 5): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=07 L00: Traceback (most recent call last):
rank=07 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=07 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=07 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=07 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=07 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=07 L06:     f(args)
rank=07 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=07 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=07 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=07 L10:     return _run_module_code(code, init_globals, run_name,
rank=07 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=07 L12:     _run_code(code, mod_globals, init_globals,
rank=07 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=07 L14:     exec(code, run_globals)
rank=07 L15:   File "get_me_coupling.py", line 91, in <module>
rank=07 L16:     structure.get_potential_energy()
rank=07 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=07 L18:     energy = self._calc.get_potential_energy(self)
rank=07 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=07 L20:     return self.get_property(name, atoms)
rank=07 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=07 L22:     self.calculate(atoms, [name], system_changes)
rank=07 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=07 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=07 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=07 L26:     yield from self.scf.irun(
rank=07 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=07 L28:     raise KohnShamConvergenceError(
rank=07 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 7): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=09 L00: Traceback (most recent call last):
rank=09 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=09 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=09 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=09 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=09 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=09 L06:     f(args)
rank=09 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=09 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=09 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=09 L10:     return _run_module_code(code, init_globals, run_name,
rank=09 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=09 L12:     _run_code(code, mod_globals, init_globals,
rank=09 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=09 L14:     exec(code, run_globals)
rank=09 L15:   File "get_me_coupling.py", line 91, in <module>
rank=09 L16:     structure.get_potential_energy()
rank=09 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=09 L18:     energy = self._calc.get_potential_energy(self)
rank=09 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=09 L20:     return self.get_property(name, atoms)
rank=09 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=09 L22:     self.calculate(atoms, [name], system_changes)
rank=09 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=09 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=09 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=09 L26:     yield from self.scf.irun(
rank=09 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=09 L28:     raise KohnShamConvergenceError(
rank=09 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 9): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=11 L00: Traceback (most recent call last):
rank=11 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=11 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=11 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=11 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=11 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=11 L06:     f(args)
rank=11 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=11 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=11 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=11 L10:     return _run_module_code(code, init_globals, run_name,
rank=11 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=11 L12:     _run_code(code, mod_globals, init_globals,
rank=11 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=11 L14:     exec(code, run_globals)
rank=11 L15:   File "get_me_coupling.py", line 91, in <module>
rank=11 L16:     structure.get_potential_energy()
rank=11 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=11 L18:     energy = self._calc.get_potential_energy(self)
rank=11 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=11 L20:     return self.get_property(name, atoms)
rank=11 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=11 L22:     self.calculate(atoms, [name], system_changes)
rank=11 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=11 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=11 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=11 L26:     yield from self.scf.irun(
rank=11 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=11 L28:     raise KohnShamConvergenceError(
rank=11 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 11): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=13 L00: Traceback (most recent call last):
rank=13 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=13 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=13 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=13 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=13 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=13 L06:     f(args)
rank=13 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=13 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=13 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=13 L10:     return _run_module_code(code, init_globals, run_name,
rank=13 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=13 L12:     _run_code(code, mod_globals, init_globals,
rank=13 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=13 L14:     exec(code, run_globals)
rank=13 L15:   File "get_me_coupling.py", line 91, in <module>
rank=13 L16:     structure.get_potential_energy()
rank=13 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=13 L18:     energy = self._calc.get_potential_energy(self)
rank=13 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=13 L20:     return self.get_property(name, atoms)
rank=13 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=13 L22:     self.calculate(atoms, [name], system_changes)
rank=13 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=13 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=13 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=13 L26:     yield from self.scf.irun(
rank=13 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=13 L28:     raise KohnShamConvergenceError(
rank=13 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 13): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=14 L00: Traceback (most recent call last):
rank=14 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=14 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=14 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=14 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=14 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=14 L06:     f(args)
rank=14 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=14 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=14 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=14 L10:     return _run_module_code(code, init_globals, run_name,
rank=14 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=14 L12:     _run_code(code, mod_globals, init_globals,
rank=14 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=14 L14:     exec(code, run_globals)
rank=14 L15:   File "get_me_coupling.py", line 91, in <module>
rank=14 L16:     structure.get_potential_energy()
rank=14 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=14 L18:     energy = self._calc.get_potential_energy(self)
rank=14 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=14 L20:     return self.get_property(name, atoms)
rank=14 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=14 L22:     self.calculate(atoms, [name], system_changes)
rank=14 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=14 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=14 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=14 L26:     yield from self.scf.irun(
rank=14 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=14 L28:     raise KohnShamConvergenceError(
rank=14 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 14): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=17 L00: Traceback (most recent call last):
rank=17 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=17 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=17 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=17 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=17 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=17 L06:     f(args)
rank=17 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=17 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=17 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=17 L10:     return _run_module_code(code, init_globals, run_name,
rank=17 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=17 L12:     _run_code(code, mod_globals, init_globals,
rank=17 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=17 L14:     exec(code, run_globals)
rank=17 L15:   File "get_me_coupling.py", line 91, in <module>
rank=17 L16:     structure.get_potential_energy()
rank=17 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=17 L18:     energy = self._calc.get_potential_energy(self)
rank=17 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=17 L20:     return self.get_property(name, atoms)
rank=17 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=17 L22:     self.calculate(atoms, [name], system_changes)
rank=17 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=17 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=17 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=17 L26:     yield from self.scf.irun(
rank=17 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=17 L28:     raise KohnShamConvergenceError(
rank=17 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 17): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=18 L00: Traceback (most recent call last):
rank=18 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=18 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=18 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=18 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=18 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=18 L06:     f(args)
rank=18 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=18 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=18 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=18 L10:     return _run_module_code(code, init_globals, run_name,
rank=18 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=18 L12:     _run_code(code, mod_globals, init_globals,
rank=18 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=18 L14:     exec(code, run_globals)
rank=18 L15:   File "get_me_coupling.py", line 91, in <module>
rank=18 L16:     structure.get_potential_energy()
rank=18 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=18 L18:     energy = self._calc.get_potential_energy(self)
rank=18 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=18 L20:     return self.get_property(name, atoms)
rank=18 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=18 L22:     self.calculate(atoms, [name], system_changes)
rank=18 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=18 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=18 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=18 L26:     yield from self.scf.irun(
rank=18 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=18 L28:     raise KohnShamConvergenceError(
rank=18 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 18): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=20 L00: Traceback (most recent call last):
rank=20 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=20 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=20 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=20 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=20 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=20 L06:     f(args)
rank=20 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=20 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=20 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=20 L10:     return _run_module_code(code, init_globals, run_name,
rank=20 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=20 L12:     _run_code(code, mod_globals, init_globals,
rank=20 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=20 L14:     exec(code, run_globals)
rank=20 L15:   File "get_me_coupling.py", line 91, in <module>
rank=20 L16:     structure.get_potential_energy()
rank=20 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=20 L18:     energy = self._calc.get_potential_energy(self)
rank=20 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=20 L20:     return self.get_property(name, atoms)
rank=20 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=20 L22:     self.calculate(atoms, [name], system_changes)
rank=20 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=20 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=20 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=20 L26:     yield from self.scf.irun(
rank=20 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=20 L28:     raise KohnShamConvergenceError(
rank=20 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 20): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=22 L00: Traceback (most recent call last):
rank=22 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=22 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=22 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=22 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=22 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=22 L06:     f(args)
rank=22 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=22 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=22 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=22 L10:     return _run_module_code(code, init_globals, run_name,
rank=22 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=22 L12:     _run_code(code, mod_globals, init_globals,
rank=22 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=22 L14:     exec(code, run_globals)
rank=22 L15:   File "get_me_coupling.py", line 91, in <module>
rank=22 L16:     structure.get_potential_energy()
rank=22 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=22 L18:     energy = self._calc.get_potential_energy(self)
rank=22 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=22 L20:     return self.get_property(name, atoms)
rank=22 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=22 L22:     self.calculate(atoms, [name], system_changes)
rank=22 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=22 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=22 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=22 L26:     yield from self.scf.irun(
rank=22 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=22 L28:     raise KohnShamConvergenceError(
rank=22 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 22): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=02 L00: Traceback (most recent call last):
rank=02 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=02 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=02 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=02 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=02 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=02 L06:     f(args)
rank=02 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=02 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=02 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=02 L10:     return _run_module_code(code, init_globals, run_name,
rank=02 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=02 L12:     _run_code(code, mod_globals, init_globals,
rank=02 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=02 L14:     exec(code, run_globals)
rank=02 L15:   File "get_me_coupling.py", line 91, in <module>
rank=02 L16:     structure.get_potential_energy()
rank=02 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=02 L18:     energy = self._calc.get_potential_energy(self)
rank=02 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=02 L20:     return self.get_property(name, atoms)
rank=02 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=02 L22:     self.calculate(atoms, [name], system_changes)
rank=02 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=02 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=02 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=02 L26:     yield from self.scf.irun(
rank=02 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=02 L28:     raise KohnShamConvergenceError(
rank=02 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 2): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 1): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 4): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 3): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=10 L00: Traceback (most recent call last):
rank=10 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=10 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=10 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=10 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=10 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=10 L06:     f(args)
rank=10 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=10 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=10 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=10 L10:     return _run_module_code(code, init_globals, run_name,
rank=10 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=10 L12:     _run_code(code, mod_globals, init_globals,
rank=10 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=10 L14:     exec(code, run_globals)
rank=10 L15:   File "get_me_coupling.py", line 91, in <module>
rank=10 L16:     structure.get_potential_energy()
rank=10 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=10 L18:     energy = self._calc.get_potential_energy(self)
rank=10 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=10 L20:     return self.get_property(name, atoms)
rank=10 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=10 L22:     self.calculate(atoms, [name], system_changes)
rank=10 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=10 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=10 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=10 L26:     yield from self.scf.irun(
rank=10 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=10 L28:     raise KohnShamConvergenceError(
rank=10 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 10): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=15 L00: Traceback (most recent call last):
rank=15 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=15 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=15 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=15 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=15 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=15 L06:     f(args)
rank=15 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=15 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=15 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=15 L10:     return _run_module_code(code, init_globals, run_name,
rank=15 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=15 L12:     _run_code(code, mod_globals, init_globals,
rank=15 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=15 L14:     exec(code, run_globals)
rank=15 L15:   File "get_me_coupling.py", line 91, in <module>
rank=15 L16:     structure.get_potential_energy()
rank=15 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=15 L18:     energy = self._calc.get_potential_energy(self)
rank=15 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=15 L20:     return self.get_property(name, atoms)
rank=15 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=15 L22:     self.calculate(atoms, [name], system_changes)
rank=15 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=15 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=15 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=15 L26:     yield from self.scf.irun(
rank=15 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=15 L28:     raise KohnShamConvergenceError(
rank=15 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 15): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=21 L00: Traceback (most recent call last):
rank=21 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=21 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=21 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=21 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=21 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=21 L06:     f(args)
rank=21 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=21 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=21 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=21 L10:     return _run_module_code(code, init_globals, run_name,
rank=21 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=21 L12:     _run_code(code, mod_globals, init_globals,
rank=21 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=21 L14:     exec(code, run_globals)
rank=21 L15:   File "get_me_coupling.py", line 91, in <module>
rank=21 L16:     structure.get_potential_energy()
rank=21 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=21 L18:     energy = self._calc.get_potential_energy(self)
rank=21 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=21 L20:     return self.get_property(name, atoms)
rank=21 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=21 L22:     self.calculate(atoms, [name], system_changes)
rank=21 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=21 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=21 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=21 L26:     yield from self.scf.irun(
rank=21 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=21 L28:     raise KohnShamConvergenceError(
rank=21 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 21): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=39 L00: Traceback (most recent call last):
rank=39 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=39 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=39 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=39 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=39 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=39 L06:     f(args)
rank=39 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=39 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=39 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=39 L10:     return _run_module_code(code, init_globals, run_name,
rank=39 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=39 L12:     _run_code(code, mod_globals, init_globals,
rank=39 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=39 L14:     exec(code, run_globals)
rank=39 L15:   File "get_me_coupling.py", line 91, in <module>
rank=39 L16:     structure.get_potential_energy()
rank=39 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=39 L18:     energy = self._calc.get_potential_energy(self)
rank=39 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=39 L20:     return self.get_property(name, atoms)
rank=39 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=39 L22:     self.calculate(atoms, [name], system_changes)
rank=39 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=39 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=39 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=39 L26:     yield from self.scf.irun(
rank=39 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=39 L28:     raise KohnShamConvergenceError(
rank=39 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 39): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=41 L00: Traceback (most recent call last):
rank=41 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=41 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=41 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=41 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=41 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=41 L06:     f(args)
rank=41 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=41 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=41 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=41 L10:     return _run_module_code(code, init_globals, run_name,
rank=41 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=41 L12:     _run_code(code, mod_globals, init_globals,
rank=41 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=41 L14:     exec(code, run_globals)
rank=41 L15:   File "get_me_coupling.py", line 91, in <module>
rank=41 L16:     structure.get_potential_energy()
rank=41 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=41 L18:     energy = self._calc.get_potential_energy(self)
rank=41 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=41 L20:     return self.get_property(name, atoms)
rank=41 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=41 L22:     self.calculate(atoms, [name], system_changes)
rank=41 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=41 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=41 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=41 L26:     yield from self.scf.irun(
rank=41 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=41 L28:     raise KohnShamConvergenceError(
rank=41 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 41): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=42 L00: Traceback (most recent call last):
rank=42 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=42 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=42 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=42 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=42 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=42 L06:     f(args)
rank=42 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=42 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=42 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=42 L10:     return _run_module_code(code, init_globals, run_name,
rank=42 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=42 L12:     _run_code(code, mod_globals, init_globals,
rank=42 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=42 L14:     exec(code, run_globals)
rank=42 L15:   File "get_me_coupling.py", line 91, in <module>
rank=42 L16:     structure.get_potential_energy()
rank=42 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=42 L18:     energy = self._calc.get_potential_energy(self)
rank=42 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=42 L20:     return self.get_property(name, atoms)
rank=42 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=42 L22:     self.calculate(atoms, [name], system_changes)
rank=42 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=42 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=42 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=42 L26:     yield from self.scf.irun(
rank=42 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=42 L28:     raise KohnShamConvergenceError(
rank=42 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 42): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=43 L00: Traceback (most recent call last):
rank=43 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=43 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=43 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=43 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=43 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=43 L06:     f(args)
rank=43 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=43 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=43 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=43 L10:     return _run_module_code(code, init_globals, run_name,
rank=43 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=43 L12:     _run_code(code, mod_globals, init_globals,
rank=43 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=43 L14:     exec(code, run_globals)
rank=43 L15:   File "get_me_coupling.py", line 91, in <module>
rank=43 L16:     structure.get_potential_energy()
rank=43 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=43 L18:     energy = self._calc.get_potential_energy(self)
rank=43 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=43 L20:     return self.get_property(name, atoms)
rank=43 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=43 L22:     self.calculate(atoms, [name], system_changes)
rank=43 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=43 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=43 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=43 L26:     yield from self.scf.irun(
rank=43 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=43 L28:     raise KohnShamConvergenceError(
rank=43 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 43): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=46 L00: Traceback (most recent call last):
rank=46 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=46 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=46 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=46 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=46 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=46 L06:     f(args)
rank=46 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=46 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=46 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=46 L10:     return _run_module_code(code, init_globals, run_name,
rank=46 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=46 L12:     _run_code(code, mod_globals, init_globals,
rank=46 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=46 L14:     exec(code, run_globals)
rank=46 L15:   File "get_me_coupling.py", line 91, in <module>
rank=46 L16:     structure.get_potential_energy()
rank=46 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=46 L18:     energy = self._calc.get_potential_energy(self)
rank=46 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=46 L20:     return self.get_property(name, atoms)
rank=46 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=46 L22:     self.calculate(atoms, [name], system_changes)
rank=46 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=46 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=46 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=46 L26:     yield from self.scf.irun(
rank=46 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=46 L28:     raise KohnShamConvergenceError(
rank=46 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 46): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=27 L00: Traceback (most recent call last):
rank=27 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=27 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=27 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=27 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=27 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=27 L06:     f(args)
rank=27 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=27 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=27 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=27 L10:     return _run_module_code(code, init_globals, run_name,
rank=27 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=27 L12:     _run_code(code, mod_globals, init_globals,
rank=27 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=27 L14:     exec(code, run_globals)
rank=27 L15:   File "get_me_coupling.py", line 91, in <module>
rank=27 L16:     structure.get_potential_energy()
rank=27 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=27 L18:     energy = self._calc.get_potential_energy(self)
rank=27 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=27 L20:     return self.get_property(name, atoms)
rank=27 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=27 L22:     self.calculate(atoms, [name], system_changes)
rank=27 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=27 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=27 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=27 L26:     yield from self.scf.irun(
rank=27 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=27 L28:     raise KohnShamConvergenceError(
rank=27 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 27): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=33 L00: Traceback (most recent call last):
rank=33 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=33 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=33 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=33 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=33 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=33 L06:     f(args)
rank=33 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=33 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=33 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=33 L10:     return _run_module_code(code, init_globals, run_name,
rank=33 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=33 L12:     _run_code(code, mod_globals, init_globals,
rank=33 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=33 L14:     exec(code, run_globals)
rank=33 L15:   File "get_me_coupling.py", line 91, in <module>
rank=33 L16:     structure.get_potential_energy()
rank=33 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=33 L18:     energy = self._calc.get_potential_energy(self)
rank=33 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=33 L20:     return self.get_property(name, atoms)
rank=33 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=33 L22:     self.calculate(atoms, [name], system_changes)
rank=33 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=33 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=33 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=33 L26:     yield from self.scf.irun(
rank=33 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=33 L28:     raise KohnShamConvergenceError(
rank=33 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 33): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=34 L00: Traceback (most recent call last):
rank=34 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=34 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=34 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=34 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=34 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=34 L06:     f(args)
rank=34 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=34 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=34 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=34 L10:     return _run_module_code(code, init_globals, run_name,
rank=34 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=34 L12:     _run_code(code, mod_globals, init_globals,
rank=34 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=34 L14:     exec(code, run_globals)
rank=34 L15:   File "get_me_coupling.py", line 91, in <module>
rank=34 L16:     structure.get_potential_energy()
rank=34 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=34 L18:     energy = self._calc.get_potential_energy(self)
rank=34 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=34 L20:     return self.get_property(name, atoms)
rank=34 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=34 L22:     self.calculate(atoms, [name], system_changes)
rank=34 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=34 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=34 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=34 L26:     yield from self.scf.irun(
rank=34 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=34 L28:     raise KohnShamConvergenceError(
rank=34 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
rank=35 L00: Traceback (most recent call last):
rank=35 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=35 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=35 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=35 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=35 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=35 L06:     f(args)
rank=35 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=35 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=35 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=35 L10:     return _run_module_code(code, init_globals, run_name,
rank=35 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=35 L12:     _run_code(code, mod_globals, init_globals,
rank=35 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=35 L14:     exec(code, run_globals)
rank=35 L15:   File "get_me_coupling.py", line 91, in <module>
rank=35 L16:     structure.get_potential_energy()
rank=35 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=35 L18:     energy = self._calc.get_potential_energy(self)
rank=35 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=35 L20:     return self.get_property(name, atoms)
rank=35 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=35 L22:     self.calculate(atoms, [name], system_changes)
rank=35 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=35 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=35 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=35 L26:     yield from self.scf.irun(
rank=35 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=35 L28:     raise KohnShamConvergenceError(
rank=35 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 35): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=37 L00: Traceback (most recent call last):
rank=37 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=37 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=37 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=37 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=37 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=37 L06:     f(args)
rank=37 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=37 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=37 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=37 L10:     return _run_module_code(code, init_globals, run_name,
rank=37 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=37 L12:     _run_code(code, mod_globals, init_globals,
rank=37 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=37 L14:     exec(code, run_globals)
rank=37 L15:   File "get_me_coupling.py", line 91, in <module>
rank=37 L16:     structure.get_potential_energy()
rank=37 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=37 L18:     energy = self._calc.get_potential_energy(self)
rank=37 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=37 L20:     return self.get_property(name, atoms)
rank=37 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=37 L22:     self.calculate(atoms, [name], system_changes)
rank=37 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=37 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=37 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=37 L26:     yield from self.scf.irun(
rank=37 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=37 L28:     raise KohnShamConvergenceError(
rank=37 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 37): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=24 L00: Traceback (most recent call last):
rank=24 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=24 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=24 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=24 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=24 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=24 L06:     f(args)
rank=24 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=24 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=24 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=24 L10:     return _run_module_code(code, init_globals, run_name,
rank=24 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=24 L12:     _run_code(code, mod_globals, init_globals,
rank=24 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=24 L14:     exec(code, run_globals)
rank=24 L15:   File "get_me_coupling.py", line 91, in <module>
rank=24 L16:     structure.get_potential_energy()
rank=24 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=24 L18:     energy = self._calc.get_potential_energy(self)
rank=24 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=24 L20:     return self.get_property(name, atoms)
rank=24 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=24 L22:     self.calculate(atoms, [name], system_changes)
rank=24 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=24 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=24 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=24 L26:     yield from self.scf.irun(
rank=24 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=24 L28:     raise KohnShamConvergenceError(
rank=24 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 24): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=25 L00: Traceback (most recent call last):
rank=25 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=25 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=25 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=25 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=25 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=25 L06:     f(args)
rank=25 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=25 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=25 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=25 L10:     return _run_module_code(code, init_globals, run_name,
rank=25 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=25 L12:     _run_code(code, mod_globals, init_globals,
rank=25 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=25 L14:     exec(code, run_globals)
rank=25 L15:   File "get_me_coupling.py", line 91, in <module>
rank=25 L16:     structure.get_potential_energy()
rank=25 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=25 L18:     energy = self._calc.get_potential_energy(self)
rank=25 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=25 L20:     return self.get_property(name, atoms)
rank=25 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=25 L22:     self.calculate(atoms, [name], system_changes)
rank=25 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=25 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=25 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=25 L26:     yield from self.scf.irun(
rank=25 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=25 L28:     raise KohnShamConvergenceError(
rank=25 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 25): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=26 L00: Traceback (most recent call last):
rank=26 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=26 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=26 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=26 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=26 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=26 L06:     f(args)
rank=26 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=26 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=26 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=26 L10:     return _run_module_code(code, init_globals, run_name,
rank=26 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=26 L12:     _run_code(code, mod_globals, init_globals,
rank=26 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=26 L14:     exec(code, run_globals)
rank=26 L15:   File "get_me_coupling.py", line 91, in <module>
rank=26 L16:     structure.get_potential_energy()
rank=26 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=26 L18:     energy = self._calc.get_potential_energy(self)
rank=26 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=26 L20:     return self.get_property(name, atoms)
rank=26 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=26 L22:     self.calculate(atoms, [name], system_changes)
rank=26 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=26 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=26 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=26 L26:     yield from self.scf.irun(
rank=26 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=26 L28:     raise KohnShamConvergenceError(
rank=26 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 26): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=28 L00: Traceback (most recent call last):
rank=28 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=28 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=28 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=28 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=28 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=28 L06:     f(args)
rank=28 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=28 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=28 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=28 L10:     return _run_module_code(code, init_globals, run_name,
rank=28 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=28 L12:     _run_code(code, mod_globals, init_globals,
rank=28 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=28 L14:     exec(code, run_globals)
rank=28 L15:   File "get_me_coupling.py", line 91, in <module>
rank=28 L16:     structure.get_potential_energy()
rank=28 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=28 L18:     energy = self._calc.get_potential_energy(self)
rank=28 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=28 L20:     return self.get_property(name, atoms)
rank=28 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=28 L22:     self.calculate(atoms, [name], system_changes)
rank=28 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=28 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=28 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=28 L26:     yield from self.scf.irun(
rank=28 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=28 L28:     raise KohnShamConvergenceError(
rank=28 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 28): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=29 L00: Traceback (most recent call last):
rank=29 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=29 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=29 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=29 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=29 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=29 L06:     f(args)
rank=29 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=29 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=29 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=29 L10:     return _run_module_code(code, init_globals, run_name,
rank=29 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=29 L12:     _run_code(code, mod_globals, init_globals,
rank=29 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=29 L14:     exec(code, run_globals)
rank=29 L15:   File "get_me_coupling.py", line 91, in <module>
rank=29 L16:     structure.get_potential_energy()
rank=29 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=29 L18:     energy = self._calc.get_potential_energy(self)
rank=29 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=29 L20:     return self.get_property(name, atoms)
rank=29 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=29 L22:     self.calculate(atoms, [name], system_changes)
rank=29 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=29 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=29 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=29 L26:     yield from self.scf.irun(
rank=29 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=29 L28:     raise KohnShamConvergenceError(
rank=29 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 29): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=30 L00: Traceback (most recent call last):
rank=30 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=30 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=30 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=30 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=30 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=30 L06:     f(args)
rank=30 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=30 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=30 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=30 L10:     return _run_module_code(code, init_globals, run_name,
rank=30 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=30 L12:     _run_code(code, mod_globals, init_globals,
rank=30 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=30 L14:     exec(code, run_globals)
rank=30 L15:   File "get_me_coupling.py", line 91, in <module>
rank=30 L16:     structure.get_potential_energy()
rank=30 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=30 L18:     energy = self._calc.get_potential_energy(self)
rank=30 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=30 L20:     return self.get_property(name, atoms)
rank=30 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=30 L22:     self.calculate(atoms, [name], system_changes)
rank=30 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=30 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=30 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=30 L26:     yield from self.scf.irun(
rank=30 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=30 L28:     raise KohnShamConvergenceError(
rank=30 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 30): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=31 L00: Traceback (most recent call last):
rank=31 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=31 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=31 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=31 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=31 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=31 L06:     f(args)
rank=31 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=31 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=31 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=31 L10:     return _run_module_code(code, init_globals, run_name,
rank=31 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=31 L12:     _run_code(code, mod_globals, init_globals,
rank=31 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=31 L14:     exec(code, run_globals)
rank=31 L15:   File "get_me_coupling.py", line 91, in <module>
rank=31 L16:     structure.get_potential_energy()
rank=31 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=31 L18:     energy = self._calc.get_potential_energy(self)
rank=31 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=31 L20:     return self.get_property(name, atoms)
rank=31 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=31 L22:     self.calculate(atoms, [name], system_changes)
rank=31 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=31 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=31 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=31 L26:     yield from self.scf.irun(
rank=31 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=31 L28:     raise KohnShamConvergenceError(
rank=31 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 31): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=32 L00: Traceback (most recent call last):
rank=32 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=32 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=32 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=32 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=32 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=32 L06:     f(args)
rank=32 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=32 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=32 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=32 L10:     return _run_module_code(code, init_globals, run_name,
rank=32 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=32 L12:     _run_code(code, mod_globals, init_globals,
rank=32 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=32 L14:     exec(code, run_globals)
rank=32 L15:   File "get_me_coupling.py", line 91, in <module>
rank=32 L16:     structure.get_potential_energy()
rank=32 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=32 L18:     energy = self._calc.get_potential_energy(self)
rank=32 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=32 L20:     return self.get_property(name, atoms)
rank=32 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=32 L22:     self.calculate(atoms, [name], system_changes)
rank=32 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=32 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=32 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=32 L26:     yield from self.scf.irun(
rank=32 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=32 L28:     raise KohnShamConvergenceError(
rank=32 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 32): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=36 L00: Traceback (most recent call last):
rank=36 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=36 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=36 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=36 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=36 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=36 L06:     f(args)
rank=36 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=36 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=36 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=36 L10:     return _run_module_code(code, init_globals, run_name,
rank=36 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=36 L12:     _run_code(code, mod_globals, init_globals,
rank=36 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=36 L14:     exec(code, run_globals)
rank=36 L15:   File "get_me_coupling.py", line 91, in <module>
rank=36 L16:     structure.get_potential_energy()
rank=36 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=36 L18:     energy = self._calc.get_potential_energy(self)
rank=36 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=36 L20:     return self.get_property(name, atoms)
rank=36 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=36 L22:     self.calculate(atoms, [name], system_changes)
rank=36 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=36 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=36 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=36 L26:     yield from self.scf.irun(
rank=36 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=36 L28:     raise KohnShamConvergenceError(
rank=36 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 36): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=38 L00: Traceback (most recent call last):
rank=38 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=38 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=38 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=38 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=38 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=38 L06:     f(args)
rank=38 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=38 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=38 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=38 L10:     return _run_module_code(code, init_globals, run_name,
rank=38 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=38 L12:     _run_code(code, mod_globals, init_globals,
rank=38 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=38 L14:     exec(code, run_globals)
rank=38 L15:   File "get_me_coupling.py", line 91, in <module>
rank=38 L16:     structure.get_potential_energy()
rank=38 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=38 L18:     energy = self._calc.get_potential_energy(self)
rank=38 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=38 L20:     return self.get_property(name, atoms)
rank=38 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=38 L22:     self.calculate(atoms, [name], system_changes)
rank=38 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=38 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=38 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=38 L26:     yield from self.scf.irun(
rank=38 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=38 L28:     raise KohnShamConvergenceError(
rank=38 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 38): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=47 L00: Traceback (most recent call last):
rank=47 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=47 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=47 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=47 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=47 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=47 L06:     f(args)
rank=47 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=47 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=47 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=47 L10:     return _run_module_code(code, init_globals, run_name,
rank=47 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=47 L12:     _run_code(code, mod_globals, init_globals,
rank=47 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=47 L14:     exec(code, run_globals)
rank=47 L15:   File "get_me_coupling.py", line 91, in <module>
rank=47 L16:     structure.get_potential_energy()
rank=47 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=47 L18:     energy = self._calc.get_potential_energy(self)
rank=47 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=47 L20:     return self.get_property(name, atoms)
rank=47 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=47 L22:     self.calculate(atoms, [name], system_changes)
rank=47 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=47 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=47 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=47 L26:     yield from self.scf.irun(
rank=47 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=47 L28:     raise KohnShamConvergenceError(
rank=47 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 47): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=40 L00: Traceback (most recent call last):
rank=40 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=40 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=40 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=40 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=40 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=40 L06:     f(args)
rank=40 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=40 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=40 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=40 L10:     return _run_module_code(code, init_globals, run_name,
rank=40 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=40 L12:     _run_code(code, mod_globals, init_globals,
rank=40 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=40 L14:     exec(code, run_globals)
rank=40 L15:   File "get_me_coupling.py", line 91, in <module>
rank=40 L16:     structure.get_potential_energy()
rank=40 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=40 L18:     energy = self._calc.get_potential_energy(self)
rank=40 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=40 L20:     return self.get_property(name, atoms)
rank=40 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=40 L22:     self.calculate(atoms, [name], system_changes)
rank=40 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=40 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=40 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=40 L26:     yield from self.scf.irun(
rank=40 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=40 L28:     raise KohnShamConvergenceError(
rank=40 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 40): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=44 L00: Traceback (most recent call last):
rank=44 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=44 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=44 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=44 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=44 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=44 L06:     f(args)
rank=44 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=44 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=44 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=44 L10:     return _run_module_code(code, init_globals, run_name,
rank=44 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=44 L12:     _run_code(code, mod_globals, init_globals,
rank=44 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=44 L14:     exec(code, run_globals)
rank=44 L15:   File "get_me_coupling.py", line 91, in <module>
rank=44 L16:     structure.get_potential_energy()
rank=44 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=44 L18:     energy = self._calc.get_potential_energy(self)
rank=44 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=44 L20:     return self.get_property(name, atoms)
rank=44 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=44 L22:     self.calculate(atoms, [name], system_changes)
rank=44 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=44 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=44 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=44 L26:     yield from self.scf.irun(
rank=44 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=44 L28:     raise KohnShamConvergenceError(
rank=44 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 44): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
rank=45 L00: Traceback (most recent call last):
rank=45 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=45 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=45 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=45 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=45 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=45 L06:     f(args)
rank=45 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=45 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=45 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=45 L10:     return _run_module_code(code, init_globals, run_name,
rank=45 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=45 L12:     _run_code(code, mod_globals, init_globals,
rank=45 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=45 L14:     exec(code, run_globals)
rank=45 L15:   File "get_me_coupling.py", line 91, in <module>
rank=45 L16:     structure.get_potential_energy()
rank=45 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=45 L18:     energy = self._calc.get_potential_energy(self)
rank=45 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=45 L20:     return self.get_property(name, atoms)
rank=45 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 499, in get_property
rank=45 L22:     self.calculate(atoms, [name], system_changes)
rank=45 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=45 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=45 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 369, in icalculate
rank=45 L26:     yield from self.scf.irun(
rank=45 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/scf.py", line 100, in irun
rank=45 L28:     raise KohnShamConvergenceError(
rank=45 L29: gpaw.KohnShamConvergenceError: Did not converge!  See text output for help.
GPAW CLEANUP (node 45): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 34): <class 'gpaw.KohnShamConvergenceError'> occurred.  Calling MPI_Abort!
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 3 in communicator MPI_COMM_WORLD
with errorcode 42.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2193
[x111.nifl.fysik.dtu.dk:19402] 47 more processes have sent help message help-mpi-api.txt / mpi-abort
[x111.nifl.fysik.dtu.dk:19402] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
